src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -2,16 +2,78 @@
 
 import com.chopshop166.chopshoplib.outputs.SendableSpeedController;
 
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.maps.RobotMap;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import java.lang.Math;
+
+/**
+ * What does it do? When the A button is pressed- it shoots a ball. What modes
+ * does it have? Semi-Auto and DUMP. what interactions does it have with other
+ * subsystems? Asks the Indexer and 'vision' if it's ready to shoot. How is it
+ * triggered/OI? A semi-auto button A- or X for shoot all. Does it store any
+ * state? No. Sensors? No.
+ */
 
 public class Shooter extends SubsystemBase {
 
-    private SendableSpeedController shooterWheelMotor;
+    private final SendableSpeedController shooterWheelMotor;
```

Comment:
This should be `PIDSpeedController`. We'll want to set the reference point instead of the normal `set` command.